src/deadline_worker_agent/api_models.py

```diff
@@ -46,6 +46,7 @@
 EnvironmentActionType = Literal["ENV_ENTER", "ENV_EXIT"]
 StepActionType = Literal["TASK_RUN"]  # noqa
 SyncInputJobAttachmentsActionType = Literal["SYNC_INPUT_JOB_ATTACHMENTS"]  # noqa
+AttachmentDownloadActionType = Literal["SYNC_INPUT_JOB_ATTACHMENTS"]  # noqa
```

Comment:
I'd like a clean cut between the `SyncInputJobAttachmentsAction` and `AttachmentDownloadAction`. Once the new feature is released, all `SyncInputJobAttachmentsAction` related code will be cleaned up. This `AttachmentDownloadAction` is also corresponding to the  upcoming`AttachmentUploadAction`.

Reply:
So after looking again, I see some deviation in `api_models.py` from our service model schema For example, we have `SYNC_INPUT_JOB_ATTACHMENTS` here vs. `syncInputJobAttachments` in our API.

I think this may be an artifact from earlier revisions of our service model that have not been updated in the code.
Longer term, I'd like to consolidate these types and have them model our service model API shape, but I'm okay with leaving this out-of-scope for this PR.